www/components/Docs/Class.tsx

```diff
@@ -58,16 +58,16 @@ const InstanceVariables = ({ cls }: Props) => (
 
 export default function Class({ cls }: Props) {
   return (
-    <Box>
-      <hl.Container variant="h4">
+    <div>
+      <Code>
```

Comment:
I think we lost the `h4`-ness here. Maybe that's OK?

Reply:
Code doesn't have to be h4 🙂 
We did lose a few headers here and there, but those will come back in actual theming. This was just a baseline to have Emotion instead of all the other stuff.